gerrit_to_github_issues/github_issues.py

```diff
@@ -75,3 +75,52 @@ def get_bot_comment(issue: Issue, bot_name: str, ps_number: str) -> IssueComment
     for i in issue.get_comments():
         if i.user.login == bot_name and ps_number in i.body:
             return i
+
+
+def assign_issues(repo)
+    open_issues = [i for i in repo.get_issues() if i.state == 'open']
+    for issue in open_issues:
+        try_assign(issue)
+
+
+def try_assign(issue):
+    # find the most recent assignment request
+    assignment_request = None
+    for comment in issue.get_comments().reversed:
+        if '/assign' in comment.body:
+            assignment_request = comment
+            break
+    if not assignment_request:
+        # Looks like no one wants this issue
+        return
+
+    if not issue.assignees:
+        # If no one has been assigned yet, let the user take the issue
+        issue.add_to_assignees(assignment_request.user)
+        issue.create_comment(f'assigned {assignment_request.user.login}')
+        return
+
+    if issue_age(issue) > 60:
```

Comment:
2 months seems a little long, but I'm not sure if an official "stale" timeline was established

Reply:
I don't think any specific timeline has ever been officially decided on for this kind of thing, but I do agree that 2 months is a bit long. I'm pretty sure I've seen other bots which mark an issue as "stale" when it reaches a month old, so I'll decrease this to 30 days. It can be changed if we notice it's a problem.